.evergreen/config.in.yml

```diff
@@ -272,10 +277,31 @@ functions:
     - command: shell.exec
       type: test
       params:
+        shell: bash
         working_dir: src
         timeout_secs: 300
         script: |
           ${PREPARE_SHELL}
+          # Disable xtrace (just in case it was accidentally set).
```

Comment:
turns out we didn't actually run FLE tests in load balanced mode because `CSFLE_KMS_PROVIDERS` was not set